<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgChars</class>
 <widget class="QWidget" name="wdgChars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgChars</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblCharCount">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>9</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCharCountText">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>9</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>character(s) total</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="lvwChars">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnDeleteSelection">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditorPreferences">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="document-properties"/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveUp">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="go-up"/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatInterval">
          <number>25</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveDown">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="go-down"/>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
         <property name="autoRepeatInterval">
          <number>25</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnNewChar">
       <property name="text">
        <string>New character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbxCharEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Character definition</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="4" column="2" rowspan="2" colspan="2">
           <widget class="QSpinBox" name="sbxHighestPixelInFontRow">
            <property name="font">
             <font>
              <family>Consolas</family>
             </font>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QSpinBox" name="sbxLeftPixel">
            <property name="font">
             <font>
              <family>Consolas</family>
             </font>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" rowspan="2" colspan="2">
           <widget class="QLabel" name="lblHighestPixelInFontRow">
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Highest pixel in row:</string>
            </property>
            <property name="buddy">
             <cstring>sbxHighestPixelInFontRow</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="2" rowspan="2" colspan="2">
           <widget class="QSpinBox" name="sbxRightPixel">
            <property name="font">
             <font>
              <family>Consolas</family>
             </font>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="lblLeftPixel">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Leftmost pixel:</string>
            </property>
            <property name="buddy">
             <cstring>sbxLeftPixel</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="ledCharacter">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="maxLength">
             <number>1</number>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::LogicalMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="lblCharacter">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Character:</string>
            </property>
            <property name="buddy">
             <cstring>ledCharacter</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0" rowspan="2" colspan="2">
           <widget class="QLabel" name="lblRightPixel">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Rightmost pixel:</string>
            </property>
            <property name="buddy">
             <cstring>sbxRightPixel</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbxSearchChar">
       <property name="title">
        <string>Search char - implement as 'filter' like qt ui editor</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <widget class="QPushButton" name="btnNextResult">
            <property name="text">
             <string>Next Result</string>
            </property>
            <property name="shortcut">
             <string notr="true">F3</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="ledSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
             </font>
            </property>
            <property name="maxLength">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblSearch">
            <property name="text">
             <string>Search:</string>
            </property>
            <property name="buddy">
             <cstring>ledSearch</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnFind">
            <property name="text">
             <string>Find</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
